microsoft-365/commerce/billing-and-payments/manage-billing-notifications.md

```diff
@@ -64,7 +64,7 @@ Your organization's billing notifications are sent to the primary and alternate
 To change the primary and alternate email address of other admins in your organization, use the following steps.
 
 > [!NOTE]
-> Billing admins can change their own primary and alternate email addresses, but not for other admins.
+> Billing admins can only change their own alternate email address. They can not modify other admins' email addresses.
```

Comment:
Please change to:

Billing admins can only change their own alternate email address. They can't change email addresses for other admins.